fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticsRepository.java

```diff
@@ -87,6 +88,13 @@ public class StatisticsRepository {
     private static final String DROP_TABLE_STATISTICS_TEMPLATE = "DELETE FROM " + FeConstants.INTERNAL_DB_NAME
             + "." + "${tblName}" + " WHERE ${condition}";
 
+    private static final String FIND_EXPIRED_JOBS = "SELECT job_id FROM "
+            + FULL_QUALIFIED_ANALYSIS_JOB_TABLE_NAME
+            + " WHERE task_id = -1 AND ${now} - last_exec_time_in_ms  > "
```

Comment:
what's mean about `task_id = -1`

Reply:
We only need job_id here, `task_id=-1` is used to filter task item in the `analysis_jobs` table